src/views/0VizHeader/Header.vue

```diff
@@ -92,23 +93,22 @@
       >
         <rect
           class="timeline-title-box"
-          x="0"
+          x="20"
           y="-5"
           height="60"
         />
         <text
           class="text-swap"
-          x="7"
+          x="30"
           y="24"
           style="font-size: 1.3em; font-weight: 600"
         >Area burned by wildfires</text>
-
         <text
           class="timeline-title"
-          x="7"
+          x="30"
           y="44"
-          style="font-size: 1.3em; font-weight: 600"
-        >in the western U.S. from 1984 to 2020</text>
```

Comment:
I think we should keep this. There were several reviews that felt the title and annotation were disconnected, this would make that moreso.

Reply:
actually forget it, 🤷🏼